src/Schema/Builder.php

```diff
@@ -35,19 +35,13 @@ class Builder extends BaseBuilder
 
 
     /**
-     * @return list<string>
+     * @return list<array{ name: string, type: string }>
```

Comment:
This is a breaking change. Could you add this to the CHANGELOG?

Reply:
added